graphics/pixelformat.h

```diff
@@ -196,64 +212,96 @@ struct PixelFormat {
 			((b >> bLoss) << bShift);
 	}
 
+	/** Retrieve red, green, and blue values from an RGB color value. */
 	inline void colorToRGB(uint32 color, uint8 &r, uint8 &g, uint8 &b) const {
 		r = expand(rBits(), color >> rShift);
 		g = expand(gBits(), color >> gShift);
 		b = expand(bBits(), color >> bShift);
 	}
 
+	/** Retrieve alpha, red, green, and blue values from an ARGB color value. */
 	inline void colorToARGB(uint32 color, uint8 &a, uint8 &r, uint8 &g, uint8 &b) const {
 		a = (aBits() == 0) ? 0xFF : expand(aBits(), color >> aShift);
 		r = expand(rBits(), color >> rShift);
 		g = expand(gBits(), color >> gShift);
 		b = expand(bBits(), color >> bShift);
 	}
 
-	//////////////////////////////////////////////////////////////////////
-	// Convenience functions for getting number of color component bits //
-	//////////////////////////////////////////////////////////////////////
+	/**
+	 * @name Convenience functions for getting the number of color component bits
+	 * @{
+	 */
 
+	/**
+	 * Return the number of red component bits.
+	 */
 	inline byte rBits() const {
 		return (8 - rLoss);
 	}
 
+	/**
+	 * Return the number of green component bits.
+	 */
 	inline byte gBits() const {
 		return (8 - gLoss);
 	}
 
+	/**
+	 * Return the number of blue component bits.
+	 */
 	inline byte bBits() const {
 		return (8 - bLoss);
 	}
 
+	/**
+	 * Return the number of alpha component bits.
+	 */
 	inline byte aBits() const {
 		return (8 - aLoss);
 	}
 
+	/**
+	 * Return the number of component bits from the specified bits per pixel (bpp) value.
```

Comment:
Which specified `bpp` value? The function does not take any argument.
It returns the total number of bits for the pixel format.